d3a_interface/simulation_config/__init__.py

```diff
@@ -0,0 +1,11 @@
+
+
+class SimulationConfigException(Exception):
+    pass
+
+
+def validate_simulation_config(simulation_config):
+    if set(simulation_config.keys()) != \
+            {"username", "name", "uuid", "domain_name",
```

Comment:
Please use single quotes for strings whenever possible :) (see the "naming and style conventions" on Jira)